app-modules/notifications/src/Models/Contracts/CanTriggerAutoSubscription.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Assist\Notifications\Models\Contracts;
+
+interface CanTriggerAutoSubscription
```

Comment:
I wonder if because this is a contract, the name should be `TriggersAutoSubscription`, just so it doesn't sound like implementing is an option as opposed to a requirement.

Reply:
That's a good point, I am going to continue work on adding tests to this so I can change the name then